sklearn/kernel_ridge.py

```diff
@@ -94,17 +94,17 @@ class KernelRidge(MultiOutputMixin, RegressorMixin, BaseEstimator):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    sklearn.linear_model.Ridge : Linear ridge regression.
```

Comment:
Could you also add RidgeCV (in general we should encourage for the CV estimator) and the GaussianProcessRegressor (it provides an estimation of the uncertainty regarding the prediction and the kernel hyperparameter are automatically tuned without an external grid-search).

Reply:
Absolutely!